CameraOverlay/overlay_all_stats.py

```diff
@@ -135,21 +134,21 @@ def on_message(self, client, userdata, msg):
 				self.data["reed_distance"] = 0
 				self.data["count"] = 0
 		
-		elif ((topic == str(topics.DAShboard.receive_message)) & self.message_received_flag == False):
+		elif topic == str(topics.DAShboard.receive_message) and self.displaying_message == False:
 			self.message_received_time = time.time()
 			message = msg.payload.decode("utf-8")
 
 			# Display Message
 			self.message_canvas.draw_text(message, (190, self.text_height * 5), size= 1, colour=Colour.red)
 
-			# Set message_received_flag to True once a message is received
-			self.message_received_flag  = True
+			# Set displaying_message to True once a message is received
+			self.displaying_message = True
 
 		if (time.time() - self.message_received_time) >= 5:
 			self.message_canvas.clear()
 
-			# Set message_received_flag to True once displayed message is cleared
-			self.message_received_flag  = False
+			# Set displaying_message to False once displayed message is cleared
+			not self.displaying_message
```

Comment:
`not <x>` is an expression, it doesn't change the variable unless you have an assignment out front. Eg. `x = not x`